app/controllers/orga/conferences_controller.rb

```diff
@@ -33,7 +33,7 @@ def conference_params
       :round, :lightningtalkslots, :tickets, :flights, :accomodation,
       :gid, #id in orga's Google Spreadsheet (format: 2017001)
       :notes,
-      attendances_attributes: [:id, :github_handle, :_destroy]
+      conference_preferences_attributes: [:id, :github_handle, :_destroy]
```

Comment:
Having `ConferencePreferences` associated with the team rather then the user, we can also get rid of the `github_handle` here (just a tiny cleanup thing) :v: